client/fc_client.cpp

```diff
@@ -687,6 +689,44 @@ void fc_client::start_new_game()
   }
 }
 
+/**
+ * Load the modpack-installer from the start menu
+ */
+void fc_client::load_modpack()
+{
+  output_window_append(ftc_client, _("Starting the modpack installer..."));
+
+  const QString storage = freeciv_storage_dir();
+  if (storage == nullptr) {
+    output_window_append(ftc_client,
+                         _("Cannot find Freeciv21 storage directory"));
+    output_window_append(
+        ftc_client,
+        _("You'll have to start the modpack installer manually. Sorry..."));
+    return;
+  }
+
+  // Look for a modpack installer binary
+  const QString modpack_name = QStringLiteral("freeciv21-modpack-qt");
+
+  // First next to the client binary
+  // NOTE On Windows findExecutable adds the .exe automatically
+  QString location = QStandardPaths::findExecutable(
+      modpack_name, {QCoreApplication::applicationDirPath()});
+  if (location.isEmpty()) {
+    // Then in PATH
+    location = QStandardPaths::findExecutable(modpack_name);
+  }
+
+  // Start it
+  qInfo(_("Starting freeciv21-modpack-qt at %s"), qUtf8Printable(location));
+
+  QProcess *modProcess = new QProcess(this);
+  QStringList arguments;
+  modProcess->start(location, arguments);
+  modProcess->waitForFinished();
```

Comment:
We probably don't want to freeze the main window until the installer is closed, potentially triggering dead app dialogs from the system